app/views/question_flow/forms/_other_income.html.slim

```diff
@@ -3,42 +3,83 @@
+      = form.govuk_radio_buttons_fieldset :"#{type}_received",
+            legend: { text: t("question_flow.#{i18n_key}.#{type}.question") },
+            hint: { text: t("question_flow.#{i18n_key}.#{type}.question_hint") } do
+        = render "shared/radio_button_set",
+                form:,
+                field_name: :"#{type}_received",
+                buttons: [{ value: true, options: { label: { text: t("generic.yes_choice") } } },
+                          { value: false, options: { label: { text: t("generic.no_choice") } } }]
+          = render "shared/money_input",
+                  form:,
+                  field: :"#{type}_conditional_value",
+                  width: 10,
+                  label_text: t("question_flow.other_income.amount")
 
-    = render "shared/money_input",
-            form:,
-            field: :"#{type}_value",
-            width: 10,
-            label_text: t("question_flow.#{i18n_key}.#{type}.instruction"),
-            set_aria: true
+          = form.govuk_collection_radio_buttons :"#{type}_frequency",
+                                                @form.frequencies,
+                                                :first,
+                                                :last,
+                                                legend: -> { tag.p(t("question_flow.other_income.frequency")) }
+    - else
+      h2.govuk-heading-m class="govuk-!-margin-bottom-2"
+        = t("question_flow.#{i18n_key}.#{type}.label")
+      span.govuk-caption-m class="govuk-!-margin-bottom-4"
+        = t("question_flow.#{i18n_key}.#{type}.hint")
 
-    = form.govuk_collection_radio_buttons :"#{type}_frequency",
-                                          @form.frequencies,
-                                          :first,
-                                          :last,
-                                          legend: -> { tag.p(t("question_flow.#{i18n_key}.enter_frequency")) }
+      = render "shared/money_input",
+              form:,
+              field: :"#{type}_value",
+              width: 10,
+              label_text: t("question_flow.#{i18n_key}.#{type}.instruction"),
+              set_aria: true
+
+      = form.govuk_collection_radio_buttons :"#{type}_frequency",
+                                            @form.frequencies,
+                                            :first,
+                                            :last,
+                                            legend: -> { tag.p(t("question_flow.#{i18n_key}.enter_frequency")) }
 
   - OtherIncomeForm::IRREGULAR_INCOME_TYPES.each do |type|
-    h2.govuk-heading-m class="govuk-!-margin-top-9 govuk-!-margin-bottom-2"
-      = t("question_flow.#{i18n_key}.#{type}.label")
-    span.govuk-caption-m class="govuk-!-margin-bottom-4"
-      = t("question_flow.#{i18n_key}.#{type}.hint")
+    - if FeatureFlags.enabled?(:conditional_reveals, @check.session_data)
+      = form.govuk_radio_buttons_fieldset :"#{type}_received",
```

Comment:
do we need to set `aria-expanded="false"` on any of these components?

Reply:
@HettieS thankfully no - the govuk component is smart enough to detect if there's a conditional reveal attached to a radio button and add an `aria-expanded` attribute that changes value depending on whether the button's selected or not.